.github/workflows/build-and-test.yml

```diff
@@ -262,110 +282,61 @@ jobs:
         uses: actions/upload-artifact@v4
         with:
           name: smoke-test-failure-screenshots
-          path: ./pr/extensions/mssql/test-results/**/
+          path: ./extensions/mssql/test-results/**/
           retention-days: 7
 
-      - name: PR Smoke Test Report
+      - name: Smoke Test Report
         uses: dorny/test-reporter@v2
         if: success() || failure()
         with:
           name: "Smoke Test Report"
           path: ./test-reports/**/smoke-results.xml
           reporter: jest-junit
-          working-directory: ./pr/extensions/mssql
+          working-directory: ./extensions/mssql
           badge-title: "smoke-tests"
           fail-on-error: false
 
-      - name: Run smoke tests for target branch # for calculating coverage comparison
-        continue-on-error: true
-        run: |
-          export VS_CODE_VERSION=stable
-          export SERVER_NAME=localhost
-          export AUTHENTICATION_TYPE="SQL Login"
-          export USER_NAME=sa
-          export PASSWORD=${{ env.PASSWORD }}
-          export SAVE_PASSWORD=No
-          export PROFILE_NAME=test-server
-          cd target/extensions/mssql
-          export BUILT_VSIX_PATH=$(find ./ -name "*.vsix" | head -n 1)
-          DISPLAY=:10 yarn smoketest
-
       - name: Merge Smoke and Unit Test Coverage Reports
         run: |
-          if [ -f target/extensions/mssql/test/resources/mergeReports.js ]; then
-            node target/extensions/mssql/test/resources/mergeReports.js target/extensions/mssql/coverage/coverage-e2e/cobertura-coverage.xml target/extensions/mssql/coverage/cobertura-coverage.xml
+          if [ -f extensions/mssql/test/resources/mergeReports.js ]; then
```

Comment:
We should probably (eventually) move `mergeReports.js` to some shared folder in the repo root instead of within mssql.  Doesn't have to happen now, though.